internal/pkg/service/stream/storage/cleanup/node.go

```diff
@@ -111,7 +111,7 @@ func (n *Node) cleanFiles(ctx context.Context) (err error) {
 	defer func() {
 		cnt := count.Load()
 		span.SetAttributes(attribute.Int64("deletedFilesCount", cnt))
-		n.logger.With(attribute.Int64("count", cnt)).Infof(ctx, `deleted "%d" files`, cnt)
+		n.logger.With(attribute.Int64("deletedFilesCount", cnt)).Info(ctx, `deleted "<deletedFilesCount>" files`)
 	}()
```

Comment:
We want to use `Logger.With` attributes in placeholders, it was discussed multiple times.

https://github.com/keboola/keboola-as-code/pull/1600#discussion_r1497336806